### DISCUSSION TITLE
Error when launching config file

### DISCUSSION THREAD

**Discussion by fkm97** [SafeExamBrowser/seb-mac#404] [Q&A]
Hi,

We are encountering an issue with a Mac Device (version 13,2.1) using SEB version 3.1 for Mac. We are being presented with this message and am wondering if there is a solution to this error. 

![Skjermbilde 2024-05-08 kl  23 03 02](https://github.com/SafeExamBrowser/seb-mac/assets/72949739/3b26d1f6-65de-4e08-9304-7dd32e8214d1)

**Comment by danschlet** [comment]
- Does this crash happen every time you start SEB on that device? 
- Did you try if the error still happens with the current SEB version? We don't support outdated SEB versions, please always confirm if an issue also happens with the latest version.
- We recommend (as Apple does) to always use the latest macOS minor version (13.2.1 is a very old macOS Ventura version and therefore very insecure).
